relay-general/src/protocol/debugmeta.rs

```diff
@@ -23,21 +23,22 @@ use crate::types::{
 /// Those strings get special treatment in our PII processor to avoid stripping the basename.
 #[derive(Debug, FromValue, ToValue, Empty, Clone, PartialEq)]
 #[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
-pub struct NativeImagePath(pub String);
+pub struct Filepath(pub String);
+// pub struct Filepath(#[metastructure(value_type = "Filepath")] pub String);
```

Comment:
@untitaker I'm not sure what the consequences are here, but this broke.

Reply:
wups, fixed